int16 left_x
int16 right_x
float32 left_theta
float32 right_theta